openmrs/i18n/registration/locale_fr.json

```diff
@@ -89,5 +89,6 @@
   "RELATIONSHIP_GRANDPARENT": "Grandparent [FR]",
   "RELATIONSHIP_CAREGIVER": "Caregiver [FR]",
   "RELATIONSHIP_AUNT_UNCLE": "Aunt/Uncle [FR]",
-  "RELATIONSHIP_OTHER": "Autre"
+  "RELATIONSHIP_OTHER": "Autre",
+  "RELATIONSHIP_SPOUSE": "époux"
```

Comment:
@IipperMangesh Please use Époux as the translation and add these translations also to the apps adt and clinical.

Reply:
Added translations in mentioned fields